tests/resources/test_remote_link.py

```diff
@@ -1,119 +1,122 @@
 from jira.exceptions import JIRAError
-from tests.conftest import JiraTestCase, broken_test
+from tests.conftest import JiraTestCase
 
 
-@broken_test(reason="Nothing from remote link works")
 class RemoteLinkTests(JiraTestCase):
     def setUp(self):
         JiraTestCase.setUp(self)
         self.issue_1 = self.test_manager.project_b_issue1
         self.issue_2 = self.test_manager.project_b_issue2
         self.issue_3 = self.test_manager.project_b_issue3
+        self.project_b_issue1_obj = self.test_manager.project_b_issue1_obj
 
     def test_remote_links(self):
         self.jira.add_remote_link(
-            "ZTRAVISDEB-3", globalId="python-test:story.of.horse.riding"
+            self.issue_1,
+            object={"url": "http://google.com", "title": "googlicious!"},
         )
-        links = self.jira.remote_links("QA-44")
+        links = self.jira.remote_links(self.issue_1)
         self.assertEqual(len(links), 1)
-        links = self.jira.remote_links("BULK-1")
+        self.jira.remote_link(self.issue_1, links[0].id).delete()
+        links = self.jira.remote_links(self.issue_2)
         self.assertEqual(len(links), 0)
 
-    @broken_test(reason="temporary disabled")
     def test_remote_links_with_issue_obj(self):
-        issue = self.jira.issue("QA-44")
-        links = self.jira.remote_links(issue)
+        self.jira.add_remote_link(
+            self.issue_1,
+            object={"url": "http://google.com", "title": "googlicious!"},
```

Comment:
let's make this default object a global and use it in this file

```python
DEFAULT_NEW_REMOTE_LINK_OBJECT = {"url": "http://google.com", "title": "googlicious!"}
```
```suggestion
            object=DEFAULT_NEW_REMOTE_LINK_OBJECT,
```

Reply:
No problem.